tests/zfs-tests/tests/functional/online_offline/online_offline_002_neg.ksh

```diff
@@ -30,6 +30,7 @@
 #
 
 . $STF_SUITE/include/libtest.shlib
+. $STF_SUITE/tests/functional/online_offline/online_offline.cfg
```

Comment:
It appears that almost all of the exports in the cfg file are unused.  There is no equivalent .cfg file in OpenZFS for this test files.  Adding a .cfg files makes sense when it's appropriate but I don' t think it should be mandatory.